ports/stm32/mpu.h

```diff
@@ -26,7 +26,89 @@
 #ifndef MICROPY_INCLUDED_STM32_MPU_H
 #define MICROPY_INCLUDED_STM32_MPU_H
 
-#if defined(STM32F7) || defined(STM32H7) || defined(MICROPY_HW_ETH_MDC)
+#if defined(STM32H5)
+
+#define MPU_REGION_SIG      (MPU_REGION_NUMBER0)
+#define MPU_REGION_ETH      (MPU_REGION_NUMBER1)
+
+#define ST_DEVICE_SIGNATURE_BASE (0x08fff800)
+#define ST_DEVICE_SIGNATURE_LIMIT (0x08ffffff)
+
+// STM32H5 Cortex-M33 MPU works differently from older cores.
+// Macro only takes region size in bytes, Attributes are coded in mpu_config_region().
+#define MPU_CONFIG_ETH(size) ( \
+    (size) \
+    )
```

Comment:
Please format this macro definition onto a single line.

Reply:
Done